supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -16,8 +16,6 @@ object Run {
     "https://www.googleapis.com/auth/compute"
   ).asJava
 
-  val NoAddressFieldName = "noAddress"
-
```

Comment:
hmm?

Reply:
The PAPI resource builder has an explicit `setNoAddress` method now (which was no the case yet when we first added this), so we don't need it anymore.